packages/eas-cli/src/project/publish.ts

```diff
@@ -118,6 +118,11 @@ export async function convertAssetToUpdateInfoGroupFormatAsync(
 ): Promise<PartialManifestAsset> {
   const fileSHA256 = getBase64URLEncoding(await calculateFileHashAsync(asset.path, 'sha256'));
   const contentType = asset.contentType;
+
+  let fileExtension;
+  if (asset.type) {
+    fileExtension = asset.type.startsWith('.') ? asset.type : '.' + asset.type;
```

Comment:
not familiar with the design here but it seems strange to me that we can store a jpg as `'.jpg'` or `'jpg`

Reply:
I agree we shouldn't.

This line is specifically meant to enforce a `.` prefix to all extension. Namely, if it has a prefix, do nothing, if it is missing a prefix, add it.

Am I misreading/going crazy? 😅 

Going to add a comment describing this as well for clarity.